seeds/cassandra/createTables.js

```diff
@@ -0,0 +1,37 @@
+const cassandra = require('cassandra-driver');
+const client = require('./config.js')
+
+
+// Create 'carsByStatusAndCategory' Table
+// Purpose: Get similar cars given some attributes of an existing car 
```

Comment:
again, not sure how you wanted to implement this but a .cql file could make all these setups with one file and one npm call... just food for thought!